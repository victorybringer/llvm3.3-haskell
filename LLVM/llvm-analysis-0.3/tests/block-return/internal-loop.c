extern int z;
int f(int x) {
  if(x < 10) {
    z += x;
    return 10;
  }

  for(int i = 0; i < x; ++i) {
    z += i - x;
  }

  return -79;
}
