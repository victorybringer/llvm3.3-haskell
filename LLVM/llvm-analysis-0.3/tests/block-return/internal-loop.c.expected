fromList [ ("%if.then", 10), ("%if.end", -79), ("%for.end", -79), ("%for.cond", -79), ("%for.body", -79) ]
